index.html

```diff
@@ -0,0 +1,776 @@
+        </p>
+        <p class="para">
+          He was trying to understand why there was no trace of some exotic
+          particles that should have been created in the Big Bang. Instead he
+          discovered what might have made the universe bang to begin with. A
+          potential hitch in the presumed course of cosmic evolution could have
+          infused space itself with a special energy that exerted a repulsive
+          force, causing the universe to swell faster than the speed of light
+          for a prodigiously violent instant.
+        </p>
+        <p class="para">
+          If true, the rapid engorgement would solve paradoxes like why the
+          heavens look uniform from pole to pole and not like a jagged, warped
+          mess. The enormous ballooning would iron out all the wrinkles and
+          irregularities. Those particles were not missing, but would be diluted
+          beyond detection, like spit in the ocean.
+        </p>
+        <p class="para">
+          “SPECTACULAR REALIZATION,” Dr. Guth wrote across the top of the page
+          and drew a double box around it.
+        </p>
+        <p class="para">
+          On Monday, Dr. Guth’s starship came in. Radio astronomers reported
+          that they had seen the beginning of the Big Bang, and that his
+          hypothesis, known undramatically as inflation, looked right.
+        </p>
+        <p class="para">
+          Reaching back across 13.8 billion years to the first sliver of cosmic
+          time with telescopes at the South Pole, a team of astronomers led by
+          John M. Kovac of the Harvard-Smithsonian Center for Astrophysics
+          detected ripples in the fabric of space-time — so-called gravitational
+          waves — the signature of a universe being wrenched violently apart
+          when it was roughly a trillionth of a trillionth of a trillionth of a
+          second old. They are the long-sought smoking-gun evidence of
+          inflation, proof, Dr. Kovac and his colleagues say, that Dr. Guth was
+          correct.
+        </p>
+        <hr class="add-hr" />
+        <p class="add"><strong>Unlock more free articles.</strong></p>
+        <p class="add"><a href="#">Create an account or log in</a></p>
+        <hr class="add-hr" />
+        <p class="para">
+          Inflation has been the workhorse of cosmology for 35 years, though
+          many, including Dr. Guth, wondered whether it could ever be proved.
+        </p>
+        <p class="para">
+          If corroborated, Dr. Kovac’s work will stand as a landmark in science
+          comparable to the recent discovery of dark energy pushing the universe
+          apart, or of the Big Bang itself. It would open vast realms of time
+          and space and energy to science and speculation.
+        </p>
+      </div>
+
+      <!-- Aside section element with Editor's Choice  -->
+
+      <div class="float">
```

Comment:
This should be an < aside > tag, not a < div >, please improve your semantic HTML :wink: